src/components/subscription-choices/subscription-choices.scss

```diff
@@ -0,0 +1,90 @@
+@use '../../compiled/tokens/scss/breakpoint';
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/a11y';
+@use '../../mixins/theme';
+
+///
+/// Theme properties
+///
+
+// Light theme
+@include theme.props {
+  --theme-color-background-subscription-form: #{color.$base-white};
+}
+
+// Dark theme
+@include theme.props(dark) {
+  --theme-color-background-subscription-form: #{color.$brand-primary};
+}
+
+// Alternate theme
+@include theme.props(alternate) {
+  --theme-color-background-subscription-form: #{color.$base-gray-lighter};
+}
+
+///
+/// Subscription Choices
+///
+
+.c-subscription-choices {
+  position: relative;
+}
+
+.c-subscription-choices__controls-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+  margin: size.$rhythm-default 0;
+
+  .c-button-swap {
+    flex-grow: 1;
+  }
+
+  .c-button-swap__subscribe {
+    display: flex;
+  }
+
+  @media (min-width: breakpoint.$s) {
+    flex-direction: row;
+  }
+}
+
+.c-subscription-choices__control {
+  flex-grow: 1;
```

Comment:
Most guides recommend using the shorthand because browsers make some other related adjustments based on this...

```suggestion
  flex: 1;
```

Reply:
Thanks! #TIL

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/9d9b917ad2c5bcf6860d998bb8d75207bd7b5c32

Reply:
For a bit more detail:

`flex: 1` is equivalent to `flex: 1 1 0`, or `flex-grow: 1; flex-shrink: 1; flex-basis: 0;`

When you only specify `flex-grow: 1` you're leaving the default flex values of `flex-shrink: 1;` and `flex-basis: auto;`.

The difference between setting your `flex-basis` to `0` or `auto` will depend on your layout, but since troubleshooting flex layouts can be confusing, it's usually better to be explicit about the expected behavior you want by either using the shorthand, or by explicitly setting all three values.